stanford_media.info.yml

```diff
@@ -17,3 +17,4 @@ dependencies:
   - drupal:path
   - entity_usage:entity_usage
   - focal_point:focal_point
+  - inline_entity_form:inline_entity_form
```

Comment:
Did we miss this or is it really a new dependency?

Reply:
Something we missed. I found it on the bulk upload form. https://github.com/SU-SWS/stanford_media/blob/8.x-2.x/src/Form/BulkUpload.php#L318